chart/templates/_commonChecks.tpl

```diff
@@ -53,16 +51,19 @@ Return common collectors for preflights and support-bundle
 Return common analyzers for preflights and support-bundle
 */}}
 {{- define "carto.replicated.commonChecks.analyzers" }}
-  - postgres:
-      checkName: PostgreSQL is available
-      collectorName: workspace-db
+  - jsonCompare:
+      checkName: Compare JSON Example
```

Comment:
Could you add a proper name and also add the other checks extracted from Postgres?